src/NuGet.Core/NuGet.Frameworks/CompatibilityProvider.cs

```diff
@@ -48,8 +48,9 @@ public bool IsCompatible(NuGetFramework target, NuGetFramework candidate)
             // check the cache for a solution
             var cacheKey = new CompatibilityCacheKey(target, candidate);
 
-            bool? result = _cache.GetOrAdd(cacheKey, (Func<CompatibilityCacheKey, bool>)((key) 
-                => { return IsCompatibleCore(target, candidate) == true; }));
+            bool? result = _cache.GetOrAdd(cacheKey, (Func<CompatibilityCacheKey, bool>)((key)
+                =>
```

Comment:
Can we simplify this part like `bool? result = _cache.GetOrAdd(cacheKey, IsCompatibleCore(target, candidate) == true);`
Otherwise very hard to read and understand.

Reply:
This looks to me like a re-formatting of existing code, but its functionality hasn't changed at all. I'm all for simplifying syntax, but I consider it out of scope for the PR (I wouldn't complain if it's improved, but I wouldn't hold up the PR because of it)